CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
```

Comment:
Is this for internal developers or external? If it's for internal, should they be forking or cloning?

Reply:
Good catch - I think they should be cloning, otherwise they would not get test automation from CircleCI CI/CD

Reply:
updated